build/yamls/flow-visibility/base/grafana.yml

```diff
@@ -193,4 +193,4 @@ spec:
   selector:
     app: grafana
   sessionAffinity: None
-  type: LoadBalancer
```

Comment:
I agree with this change, but I think we should also mention in the documentation that users can change this Service type to LoadBalancer in the downloaded YAML, if their K8s cluster supports LoadBalancer Services

When we provide a Helm chart, this can be parameterized